code/modules/libvg/utf8.dm

```diff
@@ -0,0 +1,77 @@
+#define LIBVG_LIB "./libvg.[world.system_type == UNIX ? "so" : "dll"]"
+GLOBAL_VAR_INIT(libvg_loaded, FALSE)
+/proc/LIBVG(function, list/arguments)
+	if(fexists(LIBVG_LIB))
+		call(LIBVG_LIB, function)(arglist(arguments))
```

Comment:
So I forgot that there is a way to install a dll into byond such that unsafe worlds can run it.

checking if it exists would fail in that case even thou byond would be able to run it.

it will already runtime if the file doesn't exist, no need to check for it existing.

also don't specify `./libvg` just do `libvg`, so that one can install the dll into byond's whitelist folder if they want to.